.github/workflows/issues.yml

```
@@ -0,0 +1,65 @@
+---
+  name: Create JIRA ticket for new issues
+  
+  on:
+    issues:
+      types: [opened]
+  permissions: 
+    issues: write
+    contents: read
+  jobs:
+    jira_task:
+      name: Create Jira issue
+      runs-on: ubuntu-latest
+      steps:
+      - name: Create JIRA ticket
+        id: create
+        shell: bash
+        env:
+          ISSUE_TITLE: ${{ github.event.issue.title }}
+          ISSUE_BODY: ${{ github.event.issue.body }}
+          ISSUE_URL: ${{ github.event.issue.html_url }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+          JIRA_ASSIGNEE: ${{ secrets.ASSIGNEE_JIRA_TICKET }}
+        run: |
+
+          # Use the tr command to remove the special characters from ISSUE_TITLE
+          ISSUE_TITLE_NO_SPECIAL_CHARS=$(echo "$ISSUE_TITLE" | tr -d '!$&*@#"\047\140')
+          json_response=$(curl --request POST \
+            --url 'https://jira.mongodb.org/rest/api/2/issue' \
+            --header 'Authorization: Bearer '"${JIRA_API_TOKEN}" \
+            --header 'Accept: application/json' \
+            --header 'Content-Type: application/json' \
+            --data '{
+              "fields": {
+                  "project": {
+                      "id": "19383"
```

this project is INTMDB, you need to get the id for CLOUDP

---

The cloudp ID is `10984`
```suggestion
                      "id": "10984"
```